platform_versioned_docs/version-24.1/compute-envs/aws-batch.mdx

```diff
@@ -146,7 +146,7 @@ Batch Forge automatically creates resources that you may be charged for in your
 1. Set the **Config mode** to **Batch Forge**.
 1. Select a **Provisioning model**. In most cases, this will be **Spot**. You can specify an allocation strategy and instance types under [**Advanced options**](#advanced-options). If advanced options are omitted, Seqera Platform 23.2 and later versions default to `BEST_FIT_PROGRESSIVE` for on-demand and `SPOT_CAPACITY_OPTIMIZED` for spot compute environments.
     :::note
-    You can create a compute environment that launches either spot or on-demand instances. Spot instances can cost as little as 20% of on-demand instances, and with Nextflow's ability to automatically relaunch failed tasks, spot is almost always the recommended provisioning model. Note, however, that when choosing spot instances, Seqera will also create a dedicated queue for running the main Nextflow job using a single on-demand instance to prevent any execution interruptions.
+    You can create a compute environment that launches either Spot or On-Demand Instances. Spot instances can cost as little as 20% of On-Demand Instances, and with Nextflow's ability to automatically relaunch failed tasks, spot is almost always the recommended provisioning model. Note, however, that when choosing Spot Instances, Seqera will also create a dedicated queue for running the main Nextflow job using a single On-Deand Instance to prevent any execution interruptions.
```

Comment:
```suggestion
    You can create a compute environment that launches either Spot or On-Demand instances. Spot instances can cost as little as 20% of On-Demand instances, and with Nextflow's ability to automatically relaunch failed tasks, spot is almost always the recommended provisioning model. Note, however, that when choosing Spot instances, Seqera will also create a dedicated queue for running the main Nextflow job using a single On-Demand instance to prevent any execution interruptions.
```